cocos/3d/CCTerrain.h

```diff
@@ -304,6 +304,9 @@ class CC_DLL Terrain : public Node
     friend QuadTree;
     friend Chunk;
 public:
+    void setLightMap(const std::string& fileName);
+
+    void setLightDir(const Vec3& lightDir);
```

Comment:
What space should be used when specify Light direction? World space, view space, or any other one?